PubNubUnity/Assets/PubNub/Managers/TokenManager.cs

```diff
@@ -1,601 +1,393 @@
-    }
-
-    public class UserSpacePermissions : ResourcePermission{
-        public bool Write {get; set;}
-        public bool Manage {get; set;}
-        public bool Delete {get; set;}
-        public bool Create {get; set;}
-    }
-
-    public class ResourcePermission{
-        public bool Read {get; set;}
-    }
-
-    public class GrantResources{
-        //chan
-        public Dictionary<string, int> Channels {get; set;}
-        //grp
-        public Dictionary<string, int> Groups {get; set;}
-        //usr
-        public Dictionary<string, int> Users {get; set;}
-        //spc
-        public Dictionary<string, int> Spaces {get; set;}
-    }
-
-    public class PNGrantTokenDecoded{
-        //res
-        public GrantResources Resources {get; set;}
-        //pat
-        public GrantResources Patterns {get; set;}
-        //meta
-        public Dictionary<string, object> Meta {get; set;}
-        //sig
-        public byte[] Signature {get; set;}
-        //v
-        public int Version {get; set;}
-        //t
-        public long Timestamp {get; set;}
-        //ttl
-        public int TTL {get; set;} 
-    }
-
-    public class ChannelPermissionsWithToken : ResourcePermissionsWithTokenBase{
-	  public ChannelPermissions Permissions {get; set;}
-
-    }
-
-    public class GroupPermissionsWithToken: ResourcePermissionsWithTokenBase{
-	  public GroupPermissions Permissions {get; set;}
-    }
-
-    public class UserSpacePermissionsWithToken: ResourcePermissionsWithTokenBase{
-	  public UserSpacePermissions Permissions {get; set;}
-    }
-
-    public class ResourcePermissionsWithTokenBase{
-      public Int64 BitMaskPerms {get; set;}
-      public string Token {get; set;}
-      public Int64 Timestamp {get; set;}
-      public int TTL {get; set;}
-
-    }
-
-    public class GrantResourcesWithPermissions{
-        public SafeDictionary<string, ChannelPermissionsWithToken> Channels { get; set; }
-        public SafeDictionary<string, GroupPermissionsWithToken> Groups { get; set; }
-        public SafeDictionary<string, UserSpacePermissionsWithToken> Users { get; set; }
-        public SafeDictionary<string, UserSpacePermissionsWithToken> Spaces { get; set; }
-        public SafeDictionary<string, ChannelPermissionsWithToken> ChannelsPattern { get; set; }
-        public SafeDictionary<string, GroupPermissionsWithToken> GroupsPattern { get; set; }
-        public SafeDictionary<string, UserSpacePermissionsWithToken> UsersPattern { get; set; }
-        public SafeDictionary<string, UserSpacePermissionsWithToken> SpacesPattern { get; set; }
-    }
-
-    public class TokenManager
-    {
-        private GrantResourcesWithPermissions Tokens;
-        private PubNubUnity PubNubInstance { get; set;}
-        public TokenManager(PubNubUnity pn){
-            Tokens = InitGrantResourcesWithPermissions();
-            PubNubInstance = pn;
-        }
-
-        public void CleanUp() {
-            Tokens = null; 
-        }
+// using System;
```

Comment:
commented code? Can we remove unwanted code?

Reply:
deleting the whole file as it is not required anymore.